src/html/structure/modules/infragistics.ui.financialchart.templates.html

```diff
@@ -29,12 +29,6 @@
 		</div>
 	</button>
 
-	<div class="ui-ChartButtonWrapepr">
-		<span class="ui-icon ui-icon-IndicatorsIcon"></span>
-		<span class="financialChartIndicatorMenulabel">Indicators</span>
-		<span class="ui-icon-toolbar-dd"></span>
```

Comment:
ui-icon-toolbar-dd looks like the correct class list for this element.  however, the same element in the indicators menu has ui-icon and ui-icon-toolbar-dd, which i think is wrong.